src/components/overlay/Overlay.jsx

```diff
@@ -1,20 +1,37 @@
 // @flow strict
 
-import React from 'react';
 import type {Node} from 'react';
+import React from 'react';
 import classnames from 'classnames';
 
 type PropsType = {
   children?: Node,
   partial?: boolean,
   className?: string,
+  color?: ColorType,
   ...
 };
 
-const Overlay = ({partial, children, className, ...props}: PropsType) => {
+export const COLOR = {
+  BLUE: 'blue',
+  DARK: 'dark',
+};
+
+type ColorType = $Values<typeof COLOR>;
+
+const Overlay = ({
+  partial,
+  children,
+  className,
+  color,
```

Comment:
`color = "blue"`, but i see that the fallback is anyway to blue, so maybe not needed there

Reply:
ye I left fallback like it was